scene/animation/animation_tree.cpp

```diff
@@ -185,6 +186,18 @@ float AnimationNode::_blend_node(const StringName &p_subpath, const Vector<Strin
 		p_node->blends.resize(blend_count);
 	}
 
+	// If an animation node sets use_blend to false it must remain false
+	// this is a way to make it pass down the tree. Without this
+	// the next node, which will often set use_blend to true, will
+	// override our request for use_blend false.
+	if (p_use_blend) {
+		// use self
+		p_node->use_blend = this->use_blend;
+	} else {
+		// use parameter
+		p_node->use_blend = p_use_blend;
+	}
+
```

Comment:
I hate this hack. If anyone can find a better way, please make a PR / comment.
I am trying to pass a variable from an animation node to _process_graph, where the blending occurs. 
How this works: 
1. sub/add2 animation node calls blend_input
2. blend_input calls blend_node
3. blend_node sets use_blend in the input animation node
4. animation node leaf (animation/blendspace) adds animation state to the blend tree, adding its use_blend variable also.

The issue: In between steps 3. and 4. we may travel through many other nodes and the use_blend flag will be overwritten. This code segment will avoid overwriting use_blend with true if it is already false. Allowing our modified use_blend to pass down the tree and into _process_graph.

There has to be an easier way. I tried this way because I tried to modify the least amount of code.

Reply:
Instead just use: 
```cpp
p_node->use_blend = p_use_blend && use_blend;
```
(No `this` needed either)